packages/web-app-files/src/components/SideBar/Links/PublicLinks/LinkEdit.vue

```diff
@@ -190,7 +190,7 @@ export default {
   },
   computed: {
     ...mapGetters('Files', ['highlightedFile']),
-    ...mapGetters(['getToken', 'capabilities']),
+    ...mapGetters(['getToken', 'capabilities', 'isOcis']),
```

Comment:
The `isOcis` helper has been removed in https://github.com/owncloud/web/pull/6619 in favour of checking the resharing capability - do you mind looking into the usage of that? it needs to be used in a `setup` call (composition api).

Reply:
I do not understand what you asked, sorry. I've replaced with just a check from a capability.